src/components/Dropdown/Dropdown.tsx

```diff
@@ -296,7 +296,7 @@ export const Dropdown: FC<DropdownProps> = React.memo(
           }, NO_ANIMATION_DURATION);
           const menuButtonEvent: HTMLButtonElement =
             document.activeElement as HTMLButtonElement;
-          menuButtonEvent.click();
+          menuButtonEvent?.focus?.();
```

Comment:
I thought we use Tab to exit dropdown and also select the element we last focused in? Maybe worth checking with Swapnill too, but this looks fine to me

Reply:
There is a escape key handling for this as well